From bea76e0ec41245e1cf14bfc2fe50367c14e3aedc Mon Sep 17 00:00:00 2001
From: Allen Xu <allen.xu@freescale.com>
Date: Mon, 19 Mar 2012 13:47:38 +0800
Subject: ENGR00177082 mx6 sabreauto: enable NAND by default

The default sabreauto code enabled BT_WiFi via NAND_BT_WIFI_STEER,
change it to enable NAND as default.

Signed-off-by: Allen Xu <allen.xu@freescale.com>
---
 arch/arm/mach-mx6/board-mx6q_sabreauto.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-mx6/board-mx6q_sabreauto.c b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
index 63bd843..2d2eeed 100644
--- a/arch/arm/mach-mx6/board-mx6q_sabreauto.c
+++ b/arch/arm/mach-mx6/board-mx6q_sabreauto.c
@@ -528,7 +528,7 @@ static int max7310_u43_setup(struct i2c_client *client,
 	/*3 NAND_BT_WIFI_STEER*/
 
 	int max7310_gpio_value[] = {
-		0, 1, 0, 1, 0, 0, 0, 0,
+		0, 1, 0, 0, 0, 0, 0, 0,
 	};
 
 	int n;
-- 
1.7.9.5

